From fc1613cdb60ceef6eb1cd38c563d4ccf45c29654 Mon Sep 17 00:00:00 2001
From: Matt Ranostay <mranostay@gmail.com>
Date: Wed, 4 May 2011 11:19:43 -0500
Subject: [PATCH 1/2] Removed debug statement in a PostBuildRecorder resource

This line was outputting un-needed information to the user interface:
<P>behavior: ${instance.behavior}</P>

Signed-off-by: Matt Ranostay <mranostay@gmail.com>
---
 .../GroovyPostbuildRecorder/config.jelly           |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/src/main/resources/org/jvnet/hudson/plugins/groovypostbuild/GroovyPostbuildRecorder/config.jelly b/src/main/resources/org/jvnet/hudson/plugins/groovypostbuild/GroovyPostbuildRecorder/config.jelly
index edb7cdf..5ac4ece 100644
--- a/src/main/resources/org/jvnet/hudson/plugins/groovypostbuild/GroovyPostbuildRecorder/config.jelly
+++ b/src/main/resources/org/jvnet/hudson/plugins/groovypostbuild/GroovyPostbuildRecorder/config.jelly
@@ -23,7 +23,6 @@ THE SOFTWARE.
 -->
 
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <P>behavior: ${instance.behavior}</P>
   <f:entry title="Groovy script:">
     <f:textarea field="groovyScript" />
   </f:entry>
-- 
1.7.0.4

